### ISSUE TITLE
API example code

### ISSUE THREAD

**Issue by SIGSTACKFAULT** [cabaletta/baritone#801]
Literally write example API usage code. 

I mean, yeah, I can read [ExampleBaritoneControl.java](https://github.com/cabaletta/baritone/blob/master/src/api/java/baritone/api/utils/ExampleBaritoneControl.java) but that doesn't help much. Where do I put it? What do I use to compile it?

## Context
For example, I wanted to write code that would have baritone dig a stairwell of height *n*, stopping when it reaces a given y-level.  That seems like a sufficiently arbitrary first thing to do

## Final checklist
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by SIGSTACKFAULT** [comment]
I briefly considered opening an issue for "connect a scripting language to Baritone" but that's probably worse and already done somewhere I don't know about or something

**Comment by StijnSimons** [comment]
No need to hook a scripting language up to it. Just learn Java.

**Comment by SIGSTACKFAULT** [comment]
I already know java, but don't know what to write because there's no example code, or instructions on what to do with it.

**Comment by SIGSTACKFAULT** [comment]
e.g., if I said "I want to script baritone to dig a stairwell, how do I do that?" what's the first thing you'd link me to?

**Comment by StijnSimons** [comment]
https://github.com/cabaletta/baritone

**Comment by IronException** [comment]
tbh a script is not that bad of an idea. Many people have ideas for a baritone process but know nothing about programming. For them a script might still be an ambitious task but much easier. This also allows for more flexibility. Like if you need baritone to do something simple you don't have to fork baritone and add a new process (skipping understanding everything) and then only after you built it and restarted you can use it. A script could just be added and then used while in game.

**Comment by SIGSTACKFAULT** [comment]
@IronException assuming we go with my Jython idea, I'll write the Python code if you write the Java code.

**Comment by 0-x-2-2** [comment]
cringe

**Comment by SIGSTACKFAULT** [comment]
Should I create another issue for scripting?

**Comment by SIGSTACKFAULT** [comment]
@0-x-2-2 what?

**Comment by SIGSTACKFAULT** [comment]
I'm going to create a separate issue for the scripting thing.

**Comment by 0-x-2-2** [comment]
The Example API code is ExampleBaritoneControl. Using the Baritone API it requires you to know how to set up a modded Minecraft development environment with ForgeGradle and how to properly add other compiled mods to the development environment.

>What do I use to compile it?

It also requires that you know how to compile java programs.

>I'm going to create a separate issue for the scripting thing. #881

https://github.com/cabaletta/baritone/issues/881#issuecomment-526792216
